lci_strategic_plugin/include/lci_strategic_plugin/lci_strategic_plugin_config.h

```diff
@@ -84,17 +84,17 @@ struct LCIStrategicPluginConfig
   std::string vehicle_id = "default_id";
 
   //! The name to use for this plugin during comminications with the arbitrator
-  std::string strategic_plugin_name = "LCIStrategicPlugin";
+  std::string strategic_plugin_name = "intersection_transit_maneuvering";
```

Comment:
Is this correct? This seems like more than just a rename right?

Reply:
Definitely a typo, I'll fix it.